diff --git a/node_modules/gridsome/app/fetch.js b/node_modules/gridsome/app/fetch.js
index 1d299be..100059b 100644
--- a/node_modules/gridsome/app/fetch.js
+++ b/node_modules/gridsome/app/fetch.js
@@ -107,7 +107,6 @@ function fetchJSON (jsonPath) {
     const req = new XMLHttpRequest()
 
     req.open('GET', jsonPath, true)
-    req.withCredentials = true
 
     req.onload = () => {
       switch (req.status) {
diff --git a/node_modules/gridsome/app/utils/prefetch.js b/node_modules/gridsome/app/utils/prefetch.js
index 5855536..b3ba2db 100644
--- a/node_modules/gridsome/app/utils/prefetch.js
+++ b/node_modules/gridsome/app/utils/prefetch.js
@@ -29,7 +29,6 @@ export default function prefetch (url) {
     } else {
       const req = new XMLHttpRequest()
       req.open('GET', url, true)
-      req.withCredentials = true
 
       req.onload = () => {
         if (req.status === 200) resolve()
